api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -66,19 +66,28 @@ message RBAC {
 
   // Should we do safe-list or block-list style access control?
   enum Action {
-    // The policies grant access to principals. The rest is denied. This is safe-list style
+    // The policies grant access to principals. The rest are denied. This is safe-list style
     // access control. This is the default type.
     ALLOW = 0;
 
-    // The policies deny access to principals. The rest is allowed. This is block-list style
+    // The policies deny access to principals. The rest are allowed. This is block-list style
     // access control.
     DENY = 1;
+
+    // All requests are allowed. The filter sets the `envoy.log` dynamic metadata key to "yes" if
```

Comment:
Are you sure you want this to be an action and not some boolean that is coupled with the ALLOW action?

Reply:
We want to classify which requests we should log, and that is not necessarily the same as specifying which requests to allow. If we were to combine logging with allow/deny, this would require a larger change to the API to add a separate log section to specify which permissions/principals base the logging decision on.